### DISCUSSION TITLE
Charged energy per car (per RFID)

### DISCUSSION THREAD

**Discussion by Apfelbaeckchen** [cathiele/homeassistant-goecharger#73] [Ideas]
Hi cathiele, 
first of all: Thanks a lot for your great work! 💪
My question/idea:
Is it possible to read the status of the RFID Chips and transfer it to HA? I have three cars loading on my wallbox, each with an individual RFID chip in order to keep track of the energy consumption of every car.
I would like to fetch the data für displaying them in HomeAssistant, and probably for calculating the consumption of the cars per 100km.
Any ideas how to get this done?
Thanks in advance,
Apfelbaeckchen

**Comment by BadP1t** [comment]
Hello Apfelbaeckchen, 
did you make any progress to handle this?
I´m trying to solve this, too.
